scripts/eval/eval.py

```diff
@@ -151,6 +191,10 @@ def evaluate_model(
 
     assert composer_model is not None
 
+    if mosaicml_logger is not None:
+        log_analytics_details(mosaicml_logger, model_cfg, tokenizer_name,
```

Comment:
A complication here: we call evaluate_model multiple times - so this (namely model name) could be overwritten. 

I'd maybe suggest just skipping model_name (& tokenizer name) for eval and logging for analytics a single time, outside of evaluate_model

Reply:
I'm thinking about moving this out to `main`. The only thing I'm curious about here is how this affects [certain fields that rely on the `model_cfg` variable](https://github.com/mosaicml/llm-foundry/blob/angel/log-data-for-run-analytics/scripts/eval/eval.py#L60). Since we load multiple, should I throw this information into a list like we did with `llmfoundry/eval_loaders`?

Reply:
Implemented this solution tentatively. If you think it will be useful, I can add `model_name` and `tokenizer_name` back into metrics, but for now I removed them per the comment above. @aspfohl